public/locales/ru/common.json

```diff
@@ -459,7 +459,7 @@
       "same-collection": "Рекомендуемые передачи из",
       "any-topic": "Любая тема",
       "default": "Рекомендуемые",
-      "popular": "",
+      "popular": "Наиболее просматриваемые",
```

Comment:
not mine, merged from before

Reply:
@bbfsdev Should it be empty or not?

Reply:
Mine should be empty, please duplicate this key... Make your own.